CategorySuggest.body.php

```diff
@@ -95,17 +90,13 @@ function fnCategorySuggestSaveHook( $m_isUpload, $m_pageObj ) {
 	 	foreach( $arrSelectedCats as $m_cat ) {
 	 	 	if(strlen($m_cat)>0){
 				$m_cat = Title::capitalize($m_cat, NS_CATEGORY);
-				$m_text .= "\n[[$m_catString:" . trim($m_cat) . "]]";
+				$m_text .= "\n[[". $m_catString .":" . mysql_escape_string(trim($m_cat)) . "]]";
```

Comment:
I don't really understand where that function comes from. Additionally, mysql functions are deprecated in PHP 5 too and here it has nothing to do. Removed.